### ISSUE TITLE
how do i make the bot hold left click?

### ISSUE THREAD

**Issue by giofonta289** [PrismarineJS/mineflayer#1894]
i need to make my bot farm sugar canes like this https://youtu.be/YE0jPd0EVSg?t=185

but i can't figure out a way to make it hold left click

isn't there something like .setControlState for the mouse buttons?

**Comment by DrMoraschi** [comment]
No, use bot.dig, also for questions like this, please ask on the Discord Server

**Comment by giofonta289** [comment]
> 
> 
>  please ask on the Discord Server

could you please link me the discord server? i can't find a link to it, thank you

**Comment by aesthetic0001** [comment]
https://discord.gg/SEM9r7uv

**Comment by aesthetic0001** [comment]
this should be closed i think
